starter_code/app.js

```diff
@@ -1,21 +1,37 @@
-
 const express = require('express');
-const hbs     = require('hbs');
+const expressLayouts = require('express-ejs-layouts');
 const app     = express();
 const path    = require('path');
```

Comment:
cuidado con el orden.. los require vienen arriba y solo después hacéis `const app = express()`